notebooks/recommendation_systems/labs/tfrs_basic_retrieval.ipynb

```diff
@@ -1147,7 +1147,9 @@
     "id": "15PtZqoO5k_k"
```

Comment:
<p>Either </p><p> </p><p>1) remove the commented lines &amp; corresponding code and mardown cells</p><p>2) create a kernel so that this code can run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>